dan_layer/template_builtin/templates/account/src/lib.rs

```diff
@@ -93,17 +93,6 @@ mod account_template {
             vault_mut.deposit(bucket);
         }
 
-        // #[access_rules(allow_all)]
-        // pub fn deposit_confidential(&mut self, bucket: Bucket, proof: ConfidentialProof) {
```

Comment:
Currently, we don't have `deposit_confidential` functionality (which I presume we want in the future). This means that every transfer of a confidential asset, using `withdraw_confidential`, will make disclose some information about the asset value/amount on `deposit`. Also, it seems that confidential buckets can be only created but not transferred (really, i.e., maintaining confidentiality). But again, I am presuming we will add this functionality in the future ?

Reply:
You can deposit a bucket with`deposit` whether it contains confidential resources or not. No need for a separate function.